### PR TITLE
Import script hints

### PR DESCRIPTION
**Pull request by cweiske**
Resolves: https://github.com/gerbera/gerbera/issues/2181

### CODE REVIEW

File: doc/scripting.rst [hunk 1]
```diff
@@ -1181,3 +1181,21 @@ Enable transcoding and set the correct path to the transcoding script.
...
+
+Best practices
+~~~~~~~~~~~~~~
+Writing a custom import script can be tricky.
+Here are some hints:
+
+- Gerbera needs to be restarted to pick up changes to the import script.
+- During startup, Gerbera scans all directories defined in the `autoscan`
+  configuration - this takes time.
+  Limit autoscan to a directory containing only few files.
```
Comment (KarlStraussberger) [inline doc/scripting.rst:1194]: Please add "For testing" here

**Comment by KarlStraussberger** [comment]
Thank you for improving our docs!

File: doc/scripting.rst (hunk 1 above)
Comment (KarlStraussberger) [reply doc/scripting.rst:1194]: I think you forgot to add the new content to your commit 😄

**Comment by cweiske** [comment]
I added "for Festung" to the headline.

File: doc/scripting.rst (hunk 1 above)
Comment (KarlStraussberger) [reply doc/scripting.rst:1194]: Sorry. Haven't seen that.

**Comment by cweiske** [comment]
I amended my commit and force-pushed the branch.

**Pull request merged**